examples/basics/visuals/mesh.py

```diff
@@ -76,14 +74,25 @@ def __init__(self):
         self.timer.start(0.016)
 
     def rotate(self, event):
-        self.rotation.rotate(1, (0, 1, 0))
+        # rotate with an irrational amount over each axis so there is no
+        # periodicity
```

Comment:
Ah, I don't know why I confused an axis angle representation. Thanks for that :)